### ISSUE TITLE
Running 'npm run format' in /addon/ directory causes the data/bundle.js to become unminified

### ISSUE THREAD

**Issue by pdehaan** [mozilla/testpilot#2123]
### Before:

```sh
$ git rev-parse --short HEAD # ac93297
$ pwd # /Users/pdehaan/dev/github/mozilla/testpilot/addon

$ npm run bundle

> testpilot-addon@1.0.0 bundle /Users/pdehaan/dev/github/mozilla/testpilot/addon
> cd ..; npm run addon:locales; cd addon/src; cross-env NODE_ENV=production webpack; cd ..;

> testpilot@1.0.0 addon:locales /Users/pdehaan/dev/github/mozilla/testpilot
> gulp addon-copy-locales
   ...

$ ls -lash data/bundle.js # 96K
$ wc -l data/bundle.js # 4 lines
```

### After:

```sh
$ npm run format
`--flow-parser` is deprecated. Use `--parser flow` instead.

$ ls -lash data/bundle.js # 175K
$ wc -l data/bundle.js # 5874 lines
```

**Comment by pdehaan** [comment]
Filed upstream as `https://github.com/jlongster/prettier/issues/579` in case there is some easy glob negation thing that I'm overlooking, or if we need a **prettier** fix.

The deprecated `--flow-parser` issue was fixed in #2124 PR.

**Comment by dannycoates** [comment]
Yeah, its kinda dumb that prettier runs on bundle.js but it'll never end up in the xpi like that. It'll get rebuilt and packaged minified.

WONTFIX until prettier has a better way than effing xargs
